tests/integration/bond_test.py

```diff
@@ -99,6 +100,19 @@ def bond99_with_slave(eth2_up):
         yield state
 
 
+def test_remove_existing_bond(setup_remove_bond99):
+    assert (
+        cmdlib.exec_cmd(["ip", "link", "add", BOND99, "type", "bond"])[0]
+        == cmdlib.RC_SUCCESS
+    )
+    assertlib.assert_present(BOND99, timeout=1)
+
+    with bond_interface(BOND99, [], create=False):
```

Comment:
It might be a bad idea to use the contextmanager here because it hides verification errors. But it might also be that it does not matter in this specific test case. However, it is for me a surprising behavior and makes it harder to understand the tests.